### PR TITLE
Add examples for Repository as per #874

### PR DESCRIPTION
**Pull request by vishalsodani**

### CODE REVIEW

File: doc/examples/Repository.rst [hunk 1]
```diff
@@ -8,4 +8,43 @@ Get repository topics
...
+
+.. code-block:: python
+
+    >>> repo = g.get_repo("PyGithub/PyGithub")
+    >>> open_issues = repo.get_issues(state='open')
+    >>> for issue in open_issues:
+    ...    print(issue.title)
+    ...
+    How to get public events?
+	GithubException: 500 None when using get_issues
```
Comment (sfdye) [inline doc/examples/Repository.rst]: Seems like caused by #911
But we don't want this to appear on the example, don't we?

Comment (vishalsodani) [reply doc/examples/Repository.rst]: :)

Comment (vishalsodani) [reply doc/examples/Repository.rst]: @sfdye Removed the exception issue in the example

File: doc/examples/Repository.rst [hunk 2]
```diff
@@ -8,4 +8,42 @@ Get repository topics
...
+--------------------------
+
+.. code-block:: python
+
+    >>> repo = g.get_repo("PyGithub/PyGithub")
+    >>> open_issues = repo.get_issues(state='open')
+    >>> for issue in open_issues:
+    ...    print(issue.title)
+    ...
+	How to get public events?
```
Comment (sfdye) [inline doc/examples/Repository.rst]: Ah I see, these are the titles of the issues. Very confusing 😂
Maybe just `print(issue`, it has nicer `repr` format.

File: doc/examples/Repository.rst [hunk 3]
```diff
@@ -8,4 +8,42 @@ Get repository topics
...
+
+Get all the labels of the repository
+------------------------------------
+
+.. code-block:: python
+
+    >>> repo = g.get_repo("PyGithub/PyGithub")
+    >>> labels = repo.get_labels(state='open')
+    >>> for label in labels:
+    ...    print(label.name)
```
Comment (sfdye) [inline doc/examples/Repository.rst]: Same here, just `print(label)`

File: doc/examples/Repository.rst (hunk 2 above)
Comment (vishalsodani) [reply doc/examples/Repository.rst]: @sfdye Made changes as requested.

Comment (sfdye) [reply doc/examples/Repository.rst]: You output should be something like this:
```
Issue(title="Add examples for Repository as per #874", number=917)
Issue(title="How to get public events?", number=913)
Issue(title="GithubException: 500 None when using get_issues", number=912)
```

same below

Comment (vishalsodani) [reply doc/examples/Repository.rst]: @sfdye Done.

**Comment by sfdye** [comment]
@vishalsodani Awesome, thank you very much!

**Pull request merged**